readme.md

```diff
@@ -57,61 +57,44 @@ to run as relay:
 
 **DO NOT RUN AS ELEVATED USER**, run as normal user.
 
-to run as client, run the `run-lokinet.bat` batch file as your normal user.
-
-
 ## Building
 
 Build requirements:
 
-* GNU Make
 * CMake
-* C++ 14 capable C++ compiler
+* C++ 17 capable C++ compiler
 * gcovr (if generating test coverage with gcc)
 * libuv >= 1.27.0
 * libsodium >= 1.0.18
-* libcurl
 
 ### Linux
 
 build:
 
     $ sudo apt install build-essential cmake git libcap-dev curl libuv1-dev libsodium-dev libcurl4-openssl-dev pkg-config
-    $ git clone https://github.com/loki-project/loki-network
+    $ git clone --recursive https://github.com/loki-project/loki-network
     $ cd loki-network
+    $ mkdir build 
+    $ cd build
+    $ cmake .. 
     $ make 
 
 install:
 
     $ sudo make install
 
-
-alternatively make a debian package with:
-
-    $ debuild -uc -us -b
-
-this puts the built packages in `../`
-
-
-#### Static Linux
-
-static native optimized:
-
-    $ make static STATIC_LINK=ON
-
-cross compile fully static armhf (rpi 2 and up)
-
-    $ make static STATIC_LINK=ON DOWNLOAD_SODIUM=ON TOOLCHAIN=contrib/cross/armhf.toolchain.cmake
-
 ### MacOS
 
 build:
     make sure you have cmake, libuv and xcode command line tools installed
     
-    $ git clone https://github.com/loki-project/loki-network
+    $ git clone --recursive https://github.com/loki-project/loki-network
     $ cd loki-network
-    $ make -j8
-
+    $ mkdir build 
+    $ cd build
+    $ cmake .. 
+    $ make 
```

Comment:
I'd suggest `make -j$(nproc)` here but apparently mac doesn't ship with it.  You probably have to ask Siri or something how many cores you have.